replica/database.cc

```diff
@@ -1066,17 +1064,15 @@ future<> database::remove(table& cf) noexcept {
     auto s = cf.schema();
     auto& ks = find_keyspace(s->ks_name());
     cf.deregister_metrics();
-    _tables_metadata._column_families.erase(s->id());
+    co_await _tables_metadata.remove_cf(s->id(), std::make_pair(s->ks_name(), s->cf_name()));
```

Comment:
Why not pass just the schema_ptr
and let the function extract its id, ks_name and cf_name?

Reply:
also, please rename to `{add,remove}_table` so we don't carry the outdated `cf` abbreviations forever.